src/components/kubectl/logs.ts

```diff
@@ -86,7 +88,15 @@ function getLogsForContainer(
         return;
     }
 
-    kubectl.invokeInSharedTerminal(cmd);
+    const resource = `${podSummary.namespace}/${podSummary.name}`;
+    const panel = LogsPanel.createOrShow('TODO', 'Loading \u25CC', resource);
+
+    kubectl.invokeAsync(cmd).then(
+        (result: ShellResult) => {
+            panel.setInfo(result.stdout, resource);
```

Comment:
Does this work with follow?  Surely `invokeAsync(...).then(...)` will show nothing until the shell invoke completes, and only _then_ send all logs (the whole of stdout) to the panel?  And `logs -f` will never complete without user intervention (which they can no longer do since there is no longer a terminal in play)?

Reply:
Nope, not currently. I need to write that code.